docs/source/onedal/algorithms/kernel-functions/rbf-kernel.rst

```diff
@@ -45,8 +45,8 @@ the problem is to compute the RBF kernel function :math:`K(x_i, y_i)` for any pa
 
 Computation method: *dense*
 ---------------------------
-The method computes the rbf kernel function :math:`K(X, Y)` for
-:math:`X` and :math:`Y` matrices.
+The method computes the rbf kernel function :math:`Z=K(X, Y), Z \in \mathbb{R}^{n \times m}` for
```

Comment:
```suggestion
The method computes the radial basis function kernel function :math:`Z=K(X, Y), Z \in \mathbb{R}^{n \times m}` for
```

Reply:
I suppose it is legal to use this abbreviation, because it has already decoded in the title:
`Radial Basis Function (RBF) kernel`.
I replaced it with capital letters.